### ISSUE TITLE
Workflow to share pictures within the photos app

### ISSUE THREAD

**Issue by serenity182** [nextcloud/photos#364]
**Is your feature request related to a problem? Please describe.**

I have the following usecase:

1.     My family is on a group event
2.     everyone takes pictues
3.     everyone uploads their photos automatially from the phone to their albums
4.     A group/shared folder for the event is created

Then it's very hard to sort this pictures into the folder.

4.     A group/shared folder for the event is created
5.     everyone can share their pictures to that folder by drag and drop from their own albums

**Describe the solution you'd like**

![Unbenannt](https://user-images.githubusercontent.com/489617/85118772-138e9e00-b221-11ea-9ee4-1a13b1fc4c5c.png)

1.     My family is on a group event
2.     everyone takes pictues
3.     everyone uploads his photos automatially from the phone to their albums
4.     A group/shared folder for the event is created
5.     everyone can share their pictures to that folder by drag and drop from their own albums

A folder-view in the sidebar (as in the mockup) might get messy over time. Maybe some kind of "splitview" for folder might help. Maybe there is already some kind of view within the "files app" that could be re-used.

**Describe alternatives you've considered**

- Sorting via Files-App (still no split-view, but drag and drop available)
- Mount photo folder via webdav/... and sort it via file manager on client

**Additional context**

**Comment by ATCWen** [comment]
Personally I'd say to browse the photos and put an "party-event" tag to the photos you sorted out. Then moving all the photos under the tag to a specific album. 
Basically your family photos lack of information saying which photo is taken on the party. Other thought, as these photos all have location information, find them with locaitons at once. This would sort out photos with single photo information.

**Comment by serenity182** [comment]
This assumes that the other family members know that there is a new tag for this event. But if I create a shared folder, they will see it directly. 

Just as a side note, many users I know are uncomfortable with the concept of tags: either a photo is in one folder, or in the other.

**Comment by dlauper** [comment]
I agree, that sharing albums (identical to photo folder) is not optimal. The workflow i picture would be like this:
- assume my photos parent is /Files/Multimedia/Photos, serenity182s is under /Photos
- serenity182 and me go on holidays. Both take pictures, and I create an album under /Files/Multimedia/Photos/2020/Dream-vacation
- I set serenity182 either as co-owner of the album (full access) or as members (upload only) or as guests (read only)
- now, the big difference to status quo: instead of just linking to album folder to serenity182 shared folder, which gets extremely messy over time, it is linked to his photos folder, maintaining my subfolders generated. He would see my pictures under  /Photos/2020/Dream-vacation and can add his. On a file system level, all photos only exist once and are hardlinked to the other datasets.

Cherry on top would be, if he's able to move it around wherever he likes. As far as I know this is possible as long as you don't try to move it to an external storage. Which, unfortunately, for me is pretty bad since everything is a mounted dataset (FreeNAS with a Nextcloud Jail).

**Comment by jakobroehrl** [comment]
Sould be solved with shared albums in Photos 2.0?
